packages/permissions/src/utils/has-permissions.ts

```diff
@@ -241,9 +241,9 @@ export const hasAppliedDataFence = (options: TOptionsForAppliedDataFence) => {
   const hasAppliedManagePermission = options.demandedPermissions.some(
     demandedPermission => {
       if (options.actualPermissions) {
-        return hasManagePermission(
-          demandedPermission,
-          options.actualPermissions
+        return (
+          demandedPermission.startsWith('Manage') &&
```

Comment:
Can't we use `hasManagePermission` here?

Reply:
Let's leave a comment as to why it needs to check for this.